.circleci/config.yml

```diff
@@ -358,8 +358,8 @@ workflows:
             mkdir -p test-reports/itest
             cd tests/integration/records/single_db
             with-db dockerized-postgres nosetests --with-xunit --xunit-file=../../../../test-reports/itest/junit.xml .
-          requires:
-            - redshift-itest
+#          requires:  # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker